### PR TITLE
I18n assets

### PR DESCRIPTION
**Pull request by Ealdwulf**
This PR adds function to find images with internationalised text. Hopefully we shouldn't need to use it often as in most cases we can get the app to composite text on top.

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+kano-i18n (3.15-1) unstable; urgency=low
```
Comment (tombettany) [inline debian/changelog]: `3.15-0`?

File: kano_i18n/assets.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+#
+# Utility function for finding non-text translated assets
+#
+
+import locale
+import os
+
+
+def get_path(base, relative, default='en_US'):
+    (code, _) = locale.getdefaultlocale()
```
Comment (tombettany) [inline kano_i18n/assets.py]: Given that this function will almost certainly be called from an i18n app, `_` will be defined as the translate function so might be best not to overwrite it.

File: kano_i18n/assets.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# Utility function for finding non-text translated assets
+#
+
+import locale
+import os
+
+
+def get_path(base, relative, default='en_US'):
```
Comment (radujipa) [inline kano_i18n/assets.py]: I would add a docstring to this explaining where i18n assets would have to be installed, params, etc

File: kano_i18n/assets.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# assets.py
+#
+# Copyright (C) 2015 Kano Computing Ltd.
```
Comment (tombettany) [inline kano_i18n/assets.py]: 2018

Comment (radujipa) [inline kano_i18n/assets.py]: *2018

File: debian/changelog (hunk 1 above)
Comment (radujipa) [inline debian/changelog]: *3.15.0

Comment (Ealdwulf) [reply debian/changelog]: Apparently that usually starts at 1 (hence all the previous ones did)
```It is conventional to restart the debian_revision at 1 each time the upstream_version is increased.

The package management system will break the version number apart at the last hyphen in the string (if there is one) to determine the upstream_version and debian_revision. The absence of a debian_revision is equivalent to a debian_revision of 0.```

File: kano_i18n/assets.py (hunk 2 above)
Comment (Ealdwulf) [reply kano_i18n/assets.py]: Excellent point.

File: kano_i18n/assets.py (hunk 3 above)
Comment (Ealdwulf) [reply kano_i18n/assets.py]: Done.

File: debian/changelog (hunk 1 above)
Comment (Ealdwulf) [reply debian/changelog]: Oh, yeah, we want a third digit. I'll add one.

**Pull request merged**